kusto/ingest/status.go

```diff
@@ -5,9 +5,9 @@ import (
 	"time"
 
 	"github.com/Azure/azure-kusto-go/kusto/ingest/internal/properties"
+	storageuid "github.com/gofrs/uuid"
 	"github.com/google/uuid"
 	"github.com/kylelemons/godebug/pretty"
-	storageuid "github.com/satori/go.uuid"
```

Comment:
I would guess this requires some code changes as IIRC there are explicit references to satori GUID in tableClient

Reply:
Changed storageuid  to point to the new package